### ISSUE TITLE
Add example to Geographic restriction

### ISSUE THREAD

**Issue by mcourtot** [EBISPOT/DUO#96]
Tiffany Boughtwood from Australian Genomics says  "particularly with some  Indigenous communities, the cultural connection to land means they want samples and data retained on country. Emphasises the importance of 'walled garden' approach for data access/analysis/'visiting'."

**Comment by MKonopko** [comment]
Per Adrian Thorogood: "discriminatory as GA4GH doesn’t want to restrict international data sharing. Use case for ex GDPR for B1MG"

However, EDI concepts should be considered.  Can there be a balance here with the idea of allowing analytical results to leave the restricted area, but the data itself must remain?

I think this may be valid for GEM Japan as well.

**Comment by MKonopko** [comment]
Per Michael Beauvais:
"A few things... I think that Tiffany's concern is valid but risks oversimplifying indigenous claims over "their" data. There is an epistemological challenge here where location/geography might be a useful heuristic, but that it doesn't fully encompass the various issues at hand.

Given the diversity of indigenous experiences and claims, this would require greater thought and working with individuals with expertise in this area. In Canada, the OCAP (ownership; control; access; possession) Principles are a dominant paradigm by which to understand issues relating to indigenous data governance. They are not specific to genomics in the way that, e.g., "Genomic Partnerships: Guidelines for genomic research with Aboriginal and Torres Strait Islander peoples of Queensland" and "Te Mata Ira: guidelines for genomic research with Māori" are.

Indeed, it may be possible to satisfy the claims of indigenous groups without geographical restrictions.

So, this is all to say, thought and expertise is required. Perhaps the best thing we can do in the short-term is to be honest about the potential limitations of the conditions within DUO w/r/t their ability to capture indigenous data governance concerns in genomics."

**Comment by solideoglori** [comment]
For future discussion.

**Comment by mcourtot** [comment]
Pinar says they have datasets that can not leave the EU so need a geo code. Georg confirms they need this as well.

**Comment by solideoglori** [comment]
Noting summary of comments from 2.24.2021 meeting - examples given of geo restrictions in Europe are more oriented towards data residency rather than data use. Therefore, leaning toward considerations to deprecate term from the DUO (while affirming the necessity of such clauses in consent forms). Comments from others to follow.

**Comment by pinarpink** [comment]
@solideoglori agree. The GDPR case is indeed a requirement on data transfer. The current response to this requirement in Europe is the so called ["federated" models](https://www.covid19dataportal.org/federated-ega), where access to data is enabled without transfer.

**Comment by mcourtot** [comment]
In preparation for the upcoming 2025 GA4GH Connect session, https://broadinstitute.swoogo.com/ga4gh13plenary/session/3347789/duo-diligence-spring-cleaning-and-roadmapping, please indicate by October 3rd whether this ticket is still relevant and if you'd like it to be discussed with the group. It will otherwise be closed.
